protobuf/CMakeLists.txt

```diff
@@ -63,11 +63,10 @@ if(${TRITON_COMMON_ENABLE_PROTOBUF})
       $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
   )
 
-  target_compile_options(
-    proto-library PRIVATE
-    $<$<OR:$<CXX_COMPILER_ID:Clang>,$<CXX_COMPILER_ID:AppleClang>,$<CXX_COMPILER_ID:GNU>>:
-      -Wall -Wextra -Wno-unused-parameter -Werror>
-    $<$<CXX_COMPILER_ID:MSVC>:/W0 /D_WIN32_WINNT=0x0A00 /EHsc /Zc:preprocessor>
```

Comment:
These options are responsible for: 

[`/W0`](https://learn.microsoft.com/en-us/cpp/build/reference/compiler-option-warning-level?view=msvc-170) - provide all warnings 
[`/D_WIN32_WINNT=0x0A00`](https://learn.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt?view=msvc-170) - defines version of Windows that binary compiled for. 
[`/EHsc`](https://learn.microsoft.com/en-us/cpp/build/reference/eh-exception-handling-model?view=msvc-170) - specific flag to manage exceptions
[`/Zc:preprocessor`](https://learn.microsoft.com/en-us/cpp/build/reference/zc-preprocessor?view=msvc-170) - conformance flag for C++ language standards.